magento/magento2
# Fixes customer edit issue when there is a newsletter queue associated with it.

### Description (*)


### Related Pull Requests


### Fixed Issues (if relevant)

1. Fixes magento/magento2#30645  Customer Edit Issue when there is a newsletter queue associated with it.


### Manual testing scenarios (*)

1.  magento/magento2#30645 Customer Edit Issue when there is a newsletter queue associated with it.
2. Please check newsletter tab queue grid functionality on the customer edit page

### Questions or comments


### Contribution checklist (*)
